indico/modules/auth/controllers.py

```diff
@@ -224,9 +246,21 @@ def _get_verified_email(self):
             return None
         return secure_serializer.loads(request.args['token'], max_age=3600, salt='register-email')
 
+    def _get_approved_email(self):
+        if 'approval_token' not in request.args:
+            return None
+        return secure_serializer.loads(request.args['approval_token'], salt='approve-email-salt')
```

Comment:
all the other places where we use `secure_serializer` don't add `-salt` in the salt string